docker/shared-build-env.dockerfile

```diff
@@ -35,25 +35,27 @@ WORKDIR /workspace
 
 # Install Gradle wrapper
 RUN mkdir -p src/webmodels/
-COPY ./src/webmodels/gradlew src/webmodels/
-COPY ./src/webmodels/gradle/ src/webmodels/gradle/
+COPY ./app/src/webmodels/gradlew src/webmodels/
```

Comment:
These changes are due to the fact that it was convenient to move the `docker` subdirectory up to the top level (previously at `app/docker`)